phone/src/locale/no.json

```diff
@@ -106,41 +106,41 @@
         "RETWEET_EXISTS": "Du har allerede retweetet denne!"
       },
       "MESSAGES": {
-        "NEW_RETWEET_BROADCAST": "@{{ profile_name }} retweetet noen!",
-        "NEW_BROADCAST": "Ny tweet fra @{{ profile_name }}!",
+        "NEW_RETWEET_BROADCAST": "@{{ profile_name }} retweetet",
+        "NEW_BROADCAST": "Ny tweet fra @{{ profile_name }}",
         "UNREAD_MESSAGES": "Du har {{ count }} nye tweets i tidslinjen din!"
       },
-      "SUBMIT_IMAGE": "Legg til media",
+      "SUBMIT_IMAGE": "Legg til",
       "TWEET": "Tweet",
       "REPORTED": " Rapportert",
       "REPORT": "Rapporter",
-      "IMAGE_PLACEHOLDER": "En link til et bilde eller GIF",
+      "IMAGE_PLACEHOLDER": "Lenke til et bilde eller GIF",
       "EDIT_DEFAULT_PROFILE_NAME": "Velg profilnavn",
-      "EDIT_PROFILE_AVATAR": "Avatar URL",
+      "EDIT_PROFILE_AVATAR": "Profilbilde",
       "EDIT_PROFILE_NAME": "Profilnavn",
       "EDIT_PROFILE_BIO": "Bio",
-      "EDIT_PROFILE_LOCATION": "Lokasjon",
+      "EDIT_PROFILE_LOCATION": "Område",
       "EDIT_PROFILE_JOB": "Jobb",
       "RETWEETED": "Retweetet",
-      "TWEET_MESSAGE_CHAR_LIMIT": "Meldingen er over karaktergrensen",
-      "TWEET_MESSAGE_NEW_LINE_LIMIT": "Meldingen er over linjegrensen",
-      "TWEET_MESSAGE_PLACEHOLDER": "Hva skjera brur?",
+      "TWEET_MESSAGE_CHAR_LIMIT": "Tweeten er for lang",
+      "TWEET_MESSAGE_NEW_LINE_LIMIT": "Tweeten har for mange linjer",
+      "TWEET_MESSAGE_PLACEHOLDER": "Hva skjer?",
       "SEARCH_TWEETS_PLACEHOLDER": "Søk etter melding eller profilnavn",
       "SEARCH_TWEETS": "Søk tweets",
-      "TIME_JUST_NOW": "Akkuratt nå",
-      "TIME_MINUTES": "m",
-      "TIME_HOURS": "h",
-      "TIME_DAYS": "d"
+      "TIME_JUST_NOW": "Akkurat nå",
+      "TIME_MINUTES": " min",
+      "TIME_HOURS": " timer",
+      "TIME_DAYS": " dager"
```

Comment:
Is it correct with the space here?

Reply:
Think it's more correct to have the space. In the english translation, timestamps are shown in one word (1m, 1h, 1d), but in norwegian you should have a space between the value and unit. Please refer to [this page](https://www.sprakradet.no/sprakhjelp/Skriveregler/Mellomrom/#nemningar) from The Language Council of Norway (Språkrådet) that says (in norwegian):

> **Det skal vere mellomrom mellom tal og (forkorta) nemningar anten nemninga står framfor talet eller etter det.**
> 10 kr, 5,68 kg, 25 l, 5 m, 10 min